clusterloader2/pkg/util/util.go

```diff
@@ -24,50 +24,102 @@ import (
 	"time"
 )
 
+var errKeyNotFound = fmt.Errorf("key not found")
```

Comment:
You want to define a type for it, e.g. like here:
https://github.com/kubernetes/kubernetes/blob/master/pkg/controller/volume/persistentvolume/scheduler_assume_cache.go#L58

Reply:
OK. Done.